src/commands/playbook/select/playbookSelect.test.ts

```diff
@@ -0,0 +1,72 @@
+import { Configuration, Logger, ServiceCollection } from "../../../models";
+import { ConfigurationService, StorageService, StoredConfigurationService } from "../../../services";
+import { CliSimulator, ServiceSimulator } from "../../../test";
+import { playbookSelectCommandFactory } from "../select";
+
+describe("Playbook Select Command", () => {
+  let configuration: Configuration;
+  let storageService: StorageService<Configuration>;
+  let logger: Logger;
+  let configurationService: ConfigurationService;
+  let serviceCollection: ServiceCollection;
+
+  beforeEach(() => {
+    configuration = {
+      playbooks: [
+        {
+          name: "active",
+          location: "",
+          isActive: true,
+        },
+        {
+          name: "inactive",
+          location: "",
+          isActive: false,
+        },
+      ],
+      projects: [],
+      appInsights: {
+        instrumentationKey: "",
+        enabled: true,
+      },
+    };
+
+    storageService = ServiceSimulator.createTestStorageService<Configuration>(configuration);
+    logger = ServiceSimulator.createTestLogger();
+    configurationService = new StoredConfigurationService(storageService, logger);
```

Comment:
I should probably use that shiny new `ServiceSimulator` method from the target branch